vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java

```diff
@@ -66,66 +61,6 @@ public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvide
         this.baseValidations = baseValidations;
     }
 
-    /**
-     * Set the context for further localization of elements to an <b>element</b> specified by the <b>name</b> and
-     * <b>state</b>
-     * @param name Any attribute or text value of the element
-     * @param state A state value of the element
-     * (<i>Possible values:</i> <b>ENABLED, DISABLED, SELECTED, NOT_SELECTED, VISIBLE, NOT_VISIBLE</b>)
-    */
-    @When("I change context to a [$state] element with the name '$name'")
-    public void changeContextToElementWithName(State state, String name)
-    {
-        resetContext();
-        WebElement element = baseValidations.assertIfElementExists(
-                String.format("An element with the name '%1$s'", name),
-                new Locator(WebLocatorType.ELEMENT_NAME, name)
-                        .addFilter(WebLocatorType.STATE, state.toString()));
-        uiContext.putSearchContext(element, () -> changeContextToElementWithName(state, name));
-    }
-
-    /**
-     * Set the context for further localization of elements to an <b>element</b>
-     * with the specified <b>attribute</b>
-     * @param attributeType A type of the element's attribute
-     * @param attributeValue A value of the element's attribute
-    */
-    @When("I change context to an element with the attribute '$attributeType'='$attributeValue'")
-    public void changeContextToElementWithAttribute(String attributeType, String attributeValue)
-    {
-        resetContext();
-
-        WebElement element = baseValidations.assertIfElementExists(
-                String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE, attributeType, attributeValue),
-                new Locator(WebLocatorType.XPATH,
-                        LocatorUtil.getXPathByAttribute(attributeType, attributeValue)));
-        uiContext.putSearchContext(element, () -> changeContextToElementWithAttribute(attributeType,
-                attributeValue));
-    }
-
-    /**
-     * Set the context for further localization of elements to an <b>element</b>
-     * with the specified <b>attribute</b> and <b>state</b>
-     * @param attributeType A type of the element's attribute
-     * @param attributeValue A value of the element's attribute
-     * @param state A state value of the element
-     * (<i>Possible values:</i> <b>ENABLED, DISABLED, SELECTED, NOT_SELECTED, VISIBLE, NOT_VISIBLE</b>)
-    */
-    @When("I change context to a [$state] element with the attribute '$attributeType'='$attributeValue'")
-    public void changeContextToStateElementWithAttribute(State state, String attributeType, String attributeValue)
-    {
-        resetContext();
-        WebElement element = baseValidations.assertIfElementExists(String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE,
```

Comment:
known issues might be broken here, I propose to keep the commit, but postpone it till `0.3.0`